src/include/uapi/ipc/header.h

```diff
@@ -109,6 +109,7 @@
 #define SOF_IPC_COMP_GET_VALUE			SOF_CMD_TYPE(0x002)
 #define SOF_IPC_COMP_SET_DATA			SOF_CMD_TYPE(0x003)
 #define SOF_IPC_COMP_GET_DATA			SOF_CMD_TYPE(0x004)
+#define SOF_IPC_COMP_EVENT_DATA			SOF_CMD_TYPE(0x005)
```

Comment:
question: is there a need for an ABI version bump?

Reply:
I expect 3 PRs with IPC updates for Keyword Detection feature: 1. notifications, 2. new IPCs for power management flow 3. large parameters. Assuming that existing IPC definition is not modified , only new messages are added ( @mmaka , could you confim for large parameters ? )  we can update SOF_ABI_MINOR for each PR. Are you ok ?

Reply:
I've bumped ABI_MINOR version.